tasks/main.yml

```diff
@@ -60,6 +60,41 @@
   tags:
     - fd_limit
 
+- name: add rabbitmqadmin config
+  template:
+    src: rabbitmqadmin.conf.j2
+    dest: /etc/rabbitmqadmin.conf
+    owner: root
+    group: root
+    mode: 600
+
+- name: add rabbitmq import/export scripts
+  copy:
+    src: "{{ item }}"
+    dest: "{{ RABBITMQ_SCRIPTS_DIR }}/{{ item }}"
+    owner: root
+    group: root
+    mode: 700
+  with_items:
+    - "{{ RABBITMQ_BACKUP_COMMAND }}"
+    - "{{ RABBITMQ_RESTORE_COMMAND }}"
+
+- name: add rabbitmq export directory
+  file:
+    path: "{{ RABBITMQ_EXPORT_DIR }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 700
+
+- name: set up cron job for rabbitmq exporting
```

Comment:
We don't need a separate cron job here.  It's easier to use [`TARSNAP_BACKUP_PRE_SCRIPT`](https://github.com/open-craft/ansible-backup-to-tarsnap/blob/45ac5b4bfd9fe4003c126c3284bb0a91ac48b86a/defaults/main.yml#L19) instead.